### PR TITLE
Add clusterName as debug info for troubleshooting

### PR DESCRIPTION
**Pull request by guessi**
### Issue

No issue, it's an minor improvement.

### Description

User may provide incorrect `--cluster-name` or modify the tag key unexpectedly, it is hard to find out if no clusterName included in error message. Having `clusterName` included in error message might be helpful for shorten the troubleshooting time spent.

### Checklist
- [ ] Added tests that cover your change (if possible)
- [ ] Added/modified documentation as required (such as the `README.md`, or the `docs` directory)
- [ ] Manually tested
- [ ] Made sure the title of the PR is a good description that can go into the release notes

### BONUS POINTS checklist: complete for good vibes and maybe prizes?! :exploding_head:
- [ ] Backfilled missing tests for code in same general area :tada:
- [ ] Refactored something and made the world a better place :star2:

### CODE REVIEW

**Review by M00nF1sh — APPROVED** [review]
/lgtm
/approve

thanks for improving the UX :D

**Comment by guessi** [comment]
It looks like kube-config for the test suite using relatively old SDK, test failed with the following message,

```
2022-06-17 17:04:45 [✖] unable to use kubectl with the EKS cluster (check 'kubectl --kubeconfig=/tmp/lb-controller-e2e/clusters/lb-controller-e2e-2696-1537833664623153152.kubeconfig version'): error: exec plugin: invalid apiVersion "client.authentication.k8s.io/v1alpha1"
```

**Comment by guessi** [comment]
@M00nF1sh I've updated the eksctl version for fixing CI test failed issue. Kindly have some time to review my second attempt for the change [commit change here](https://github.com/kubernetes-sigs/aws-load-balancer-controller/pull/2696/commits/21296c4253dab746748e93b5f26fc3f0ac322f37).

Let me know if it is required to create separated PR for the change.

File: scripts/lib/eksctl.sh [hunk 1]
```diff
@@ -76,7 +76,7 @@ eksctl::init() {
 #   node_count          EKS cluster's node count
 #   cluster_kubeconfig  filename of cluster's kubeconfig
 #
-# sample: eksctl::create_cluster awesome-cluster us-west-2 1.18 m5.xlarge 4 awesome-cluster.kubeconfig
+# sample: eksctl::create_cluster awesome-cluster us-west-2 1.19 m5.xlarge 4 awesome-cluster.kubeconfig
```
Comment (guessi) [inline scripts/lib/eksctl.sh:79]: Unrelated, simply trying to make comment aligned with actual ClusterConfig
- https://github.com/kubernetes-sigs/aws-load-balancer-controller/blob/v2.4.2/scripts/lib/eksctl_tmpl_cluster.yaml

**Pull request merged**